sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionCatalog.scala

```diff
@@ -74,9 +74,9 @@ private[sql] class HiveSessionCatalog(
   // | Methods and fields for interacting with HiveMetastoreCatalog |
   // ----------------------------------------------------------------
 
-  override def getDefaultDBPath(db: String): String = {
-    val defaultPath = context.hiveconf.getVar(HiveConf.ConfVars.METASTOREWAREHOUSE)
-    new Path(new Path(defaultPath), db + ".db").toString
+  override def getDatabasePath(dbName: String, path: Option[String] = None): String = {
+    val dbPath = path.getOrElse(context.hiveconf.getVar(HiveConf.ConfVars.METASTOREWAREHOUSE))
+    new Path(new Path(dbPath), dbName.toLowerCase() + ".db").toString
```

Comment:
@gatorsmile This function is used to create the path, not to query the path, right?